functions/classes/class.Common.php

```diff
@@ -944,6 +944,7 @@ private function get_valid_actions () {
 		return array(
 		        "add",
 		        "all-add",
+                        "copy".
```

Comment:
Replace period with comma...

Reply:
private function get_valid_actions () {
		return array(
		        "add",
		        "all-add",
                        "copy".
		        "edit",
		        "all,edit",
  

??

Reply:
This shouldn't be giving you a 500 error but is a small mistake in my typing. The period here would cause a valid action to be "copyedit". Trying to edit anything would return an Invalid Action error

Reply:
For your specific issue, my only suggestion would be to undo the changes and verify you're on v1.4

Reply:
my changes is ok?? actually am not php expert .
 return array(
"add",
"all,add",
"copy".
"edit",
"all,edit",




is that ok?

Reply:
ok let me undo

Reply:
phpIPAM IP address management [v1.4] dbversion25

Reply:
after undoing all changes that I found

Reply:
will you mind to access via anydesk ?